arangod/RocksDBEngine/RocksDBCollection.cpp

```diff
@@ -2103,11 +2098,14 @@ void RocksDBCollection::invalidateCacheEntry(RocksDBKey const& k) const {
 
 /// @brief can use non transactional range delete in write ahead log
 bool RocksDBCollection::canUseRangeDeleteInWal() const {
+  TRI_IF_FAILURE("ForceRangeDeleteInWal") { return true; }
   if (ServerState::instance()->isSingleServer()) {
-    // disableWalFilePruning is used by createIndex
-    return _numIndexCreations.load(std::memory_order_acquire) == 0;
+    return true;
```

Comment:
in single server, is it hardcoded to always use range deletion?

Reply:
Yes.